src/proof/nova.rs

```diff
@@ -295,6 +296,8 @@ mod tests {
                 assert_eq!(expected_iterations, Frame::significant_frame_count(&frames));
                 assert_eq!(adjusted_iterations, cs.len());
                 assert_eq!(expected_result, cs[cs.len() - 1].0.output.unwrap().expr);
+                let status: Status = cs[cs.len() - 1].0.output.unwrap().cont.into();
```

Comment:
Tiny, ignorable, style comment: It's probably a little clearer to bind the unwrapped content once and reuse.